packages/store/src/state-factory.ts

```diff
@@ -114,20 +119,45 @@ export class StateFactory {
     }
   }
 
+  connectActionHandlers(mappedStores: MappedStore[]): any {
+    if (this._connected) return;
+    this._actions
+      .pipe(
+        filter((ctx: ActionContext) => ctx.status === ActionStatus.Dispatched),
+        mergeMap(({ action }) => {
```

Comment:
Are we sure we want `mergeMap` here and not `concatMap`?

Reply:
for this case I don't think it really matters. as long as it isn't switchMap